setup.py: disable man page to kill txt2tags SVN checkout dependency

Signed-off-by: Steve Kenton <skenton@ou.edu>

+diff -pruN tovid-3534.ori/setup.py tovid-3534/setup.py
--- tovid-3534.ori/setup.py	2013-04-14 09:52:08.000000000 -0500
+++ tovid-3534/setup.py	2015-03-21 09:39:14.880173176 -0500
@@ -186,7 +186,7 @@ class BuildTovidInit (Command):
 
 # Build tovid-init with regular 'build' command
 build.sub_commands.append(('build_tovid_init', None))
-build.sub_commands.append(('build_docs', None))
+#build.sub_commands.append(('build_docs', None))
 
 # The actual setup
 setup(
@@ -244,8 +244,6 @@ setup(
             'src/tovid.ini',
         ]),
         # Manual page
-        ('share/man/man1',
-         ['docs/man/tovid.1']),
         # Desktop shortcut
         ('share/applications',
          ['tovidgui.desktop',
